docs/api/cases/cases-api-get-case.asciidoc

```diff
@@ -92,11 +92,15 @@ The API returns a JSON object with the retrieved case. For example:
   },
   "owner": "securitySolution",
   "severity": "low",
-  "duration": null,
+  "duration": null, <1>
   "tags": [
     "phishing",
     "social engineering",
     "bubblegum"
   ]
 }
---------------------------------------------------
\ No newline at end of file
+--------------------------------------------------
+<1> Duration represents the elapsed time from the creation of the case to its
+closure (in seconds). If the case has not been closed, the duration is set to
+`null`. If the case was closed after less than a second, the duration is rounded
+down to zero.
```

Comment:
```suggestion
down to zero.  Note: After a case is deleted, its duration data is no longer taken into account for Cases 'Average time to close' calculation. 
```

Reply:
feel free to massage that or delete it!  ^ Just adding on while we're updating it.  Is this the best place for it or do you think it belongs in the Cases 'i' tooltip in the UI?  @cnasikas @lcawl ? thanks either way!

Reply:
Since that metric is UI-specific (i.e. doesn't get returned by any of the APIs) and isn't otherwise documented, my inclination would be keep that detail in the UI tooltip.  I'll draft up a PR to see what that would look like.

Reply:
i've created https://github.com/elastic/kibana/pull/133567 for this purpose.